src/hotspot/share/classfile/vmIntrinsics.hpp

```diff
@@ -904,8 +904,8 @@ class methodHandle;
    do_name(vector_insert_name, "insert")                                                                                                       \
                                                                                                                                                \
   do_intrinsic(_VectorBroadcastInt, jdk_internal_vm_vector_VectorSupport, vector_broadcast_int_name, vector_broadcast_int_sig, F_S)            \
-   do_signature(vector_broadcast_int_sig, "(ILjava/lang/Class;Ljava/lang/Class;I"                                                              \
-                                           "Ljdk/internal/vm/vector/VectorSupport$Vector;I"                                                    \
+   do_signature(vector_broadcast_int_sig, "(ILjava/lang/Class;Ljava/lang/Class;Ljava/lang/Class;I"                                             \
+                                           "Ljdk/internal/vm/vector/VectorSupport$Vector;ILjava/lang/Object;"                                  \
```

Comment:
I notice we are inconsistent with the intrinsic declarations and the use of base types `VectorSupport.Vector/Mask` etc. for parameters and return type. I think we should clear this up in a separate PR.

Reply:
Yeah, I think the `load/store` APIs always use the base types for Vector/Mask. But for the arithmetic APIs, I remember I tried to use the same way before, but there are some issues for mask. Anyway, I will try again to clear this up. Thanks for your suggestions!